examples/immutable/timeTravelStore.res

```diff
@@ -0,0 +1,95 @@
+open SimpleStore
+
+type stringAction =
+  | A
+  | B
+
+let stringReduce = (state, action) =>
+  switch action {
+  | A => state ++ "a"
+  | B => state ++ "b"
+  }
+
+type ReduxThunk.thunk<_> +=
```

Comment:
Not sure if this is idiomatic / officially supported in ReScript: https://github.com/rescript-association/rescript-lang.org/issues/135

Reply:
I think this is supported (talking about current state) but not documented just like row polymorphism and other _advanced_ stuff.
but definitely examples need some improvement.